datasets/inquisitive/inquisitive.py

```diff
@@ -0,0 +1,166 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Inquisitive Question Generation for High Level Text Comprehension"""
+
+from __future__ import absolute_import, division, print_function
+
+import glob
+import itertools
+import os
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{ko2020inquisitive,
+  author    = {Ko, Wei-Jen and Chen, Te-Yuan and Huang, Yiyan and Durrett, Greg and Li, Junyi Jessy},
+  title     = {Inquisitive Question Generation for High Level Text Comprehension},
+  booktitle = {Proceedings of EMNLP},
+  year      = {2020},
+}
+"""
+
+_DESCRIPTION = """\
+A dataset of about 20k questions that are elicited from readers as they naturally read through a document sentence by sentence. \
+Compared to existing datasets, INQUISITIVE questions target more towards high-level (semantic and discourse) comprehension of text. \
+Because these questions are generated while the readers are pro-cessing the information, the questions directly communicate gaps between \
+the reader’s and writer’s knowledge about the events described in the text, and are not necessarily answered in the document itself. \
+This type of question reflects a real-world scenario: if one has questions during reading, some of them are answered by the text later on, \
+the rest are not, but any of them would help further the reader’s understanding at the particular point when they asked it. \
+This resource could enable question generation models to simulate human-like curiosity and cognitive processing, which may open up a new realm of applications.
+"""
+
+_ARTICLES_URL = "https://github.com/wjko2/INQUISITIVE/raw/master/articles.tgz"
+_QUESTIONS_URL = "https://github.com/wjko2/INQUISITIVE/raw/master/questions.txt"
+
+DEV_ARTICLE_IDS = list(itertools.chain(range(1, 101), range(1051, 1101)))
+TEST_ARTICLE_IDS = list(itertools.chain(range(101, 151), range(501, 551), range(1101, 1151)))
```

Comment:
Here dev and test data are not provided as separate files, instead the repo lists these ids. So hardcoding them here.
Is this the right way ?

Reply:
yes it's fine :)